include/common.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
 
 extern const char *UPS_VERSION;
 
-/* Used by upsclient and nut-scanner */
+/** @brief Default timeout (in seconds), as used by upsclient and nut-scanner. */
 #define DEFAULT_TIMEOUT 5
```

Comment:
I realize that the two timeouts (this, and the tracking timeout in `upscmd`/`upsrw`) are not at the same level of the API, but would it make sense to put the default tracking timeout here, too? At the very least, I think we should make this more specific, i.e. `DEFAULT_UPSCLIENT_TIMEOUT` or `DEFAULT_NETWORK_TIMEOUT`.